config.py

```diff
@@ -0,0 +1,145 @@
+                       'format': 'json',
+                       'cmtitle': ''}
+FILE_API_PARAMS = {'action': 'query', 'titles': '', 'prop': 'imageinfo', 'iiprop': 'url', 'format': 'json'}
+MAX_HEADER_LENGTH = 1960  # max header length as found by trail and error
+
+STYLESHEET_URL = r'https://www.reddit.com/r/dota2/about/stylesheet.json'
+FLAIR_REGEX = r'(?P<css_class>.flair-\w+),a\[href="(?P<img_path>/hero-\w+)"\]'
+RESPONSE_REGEX = r'\*(?P<files>( <sm2>.*?</sm2>)+)(?P<text>(.*))'
+CHAT_WHEEL_SECTION_REGEX = r'(=== (?P<event>The International \d+) ===)(?P<source>.+?)(?=\n=== [a-z0-9 ]+ ===\n)'
+FILE_REGEX = r'( <sm2>(?P<file>[a-zA-Z0-9_. ]+)</sm2>)'
+
+# Caching config
+CACHE_PROVIDER = os.environ.get('CACHE_PROVIDER', 'memory')  # valid choices : redis, memory, db
+CACHE_URL = os.environ.get('CACHE_URL',
+                           os.path.join(os.getcwd(), 'cache.json'))  # file path in case of memory/file based caching
+
+# DB config
+DB_PROVIDER = os.environ.get('DATABASE_PROVIDER', 'sqlite')  # valid choices : sqlite, mysql, postgres
+DB_URL = os.environ.get('DATABASE_URL', os.path.join(os.getcwd(), 'bot.db'))  # file path in case of sqlite
+
+# Logging config
+BOT_LOGGER = 'bot'
+PRAW_LOGGER = 'prawcore'
+LOG_LEVEL = os.environ.get('LOGGING_LEVEL', 'INFO').upper()
+LOG_FORMAT = '%(asctime)s %(funcName)-20s %(levelname)-8s %(message)s'
+LOG_DIR = 'logs'
+INFO_FILENAME = 'info.log'
+ERROR_FILENAME = 'error.log'
+PRAW_FILENAME = 'praw.log'
+
+CACHE_TTL = 5
+
+# Responses config
+COMMENT_ENDING = '''
+
+---
+Bleep bloop, I am a robot.
+
+[*^(Source)*](https://github.com/MePsyDuck/DotaResponsesRedditBot) *^(|)* 
+[*^(Suggestions/Issues)*](https://github.com/MePsyDuck/DotaResponsesRedditBot/issues) *^(|)* 
+[*^(Contact)*](https://www.reddit.com/user/MePsyDuck/) *^(|)* 
+[*^(Author)*](https://www.reddit.com/user/Jonarz/)
+'''
+
+# Key should be lowercase without special characters. Needs to be updated if links break (as links can be
+# non-gamepedia links too)
+# Value should have a placeholder for original text and thing ending
+CUSTOM_RESPONSES = {
+    'ho ho ha ha': '[{}](https://gamepedia.cursecdn.com/dota2_gamepedia/1/17/Snip_ability_shrapnel_03.mp3)'
+                   ' (trigger warning: Sniper){}',
+    'turn up this guy is crazy as fuck he s gotta be on molly or some powder or something':
+        '[{}](https://www.youtube.com/watch?v=CO3j9lUYFfo) (Donation warning: Arteezy){}'
+}
+
+# Only include responses for items, runes, heroes, > 100 count and common phrases.
+# Hardcoded because then they can tweaked according to the needs.
+# Drawback for this : need to update each time hero/item is added
+FREQUENT_RESPONSES = {'denied', 'yes', 'not yet', 'no mana', 'not enough mana', 'i m not ready', 'out of mana',
+                      'it s not time yet', 'ah', 'no', 'uh', 'ha ha', 'attack', 'haste', 'double damage', 'immortality',
+                      'invisibility', 'illusion', 'regeneration', 'uh uh', 'ha', }
+
+# Get them from here.
```

Comment:
Should probably be a TODO (easier to find as IDEs can list such comments then).

Reply:
Updated in 41e1416fd2c7051547d1ef4807ac17741893a239